roles/execution/vars/ufw_rules.yml

```diff
@@ -0,0 +1,7 @@
+---
+execution_ufw_rules:
+  - port: "{{ execution_p2p_port }}"
+    proto: tcp
+    rule: allow
+    from: any
+    comment: storyprotocol_execution_client_p2p_port
```

Comment:
in comment we just need to give the port variable name which is execution_p2p_port

it takes that and with other variables create a comment which will include project name too

`    comment: "{{ item.rule }} {{ env }} {{ project }} {{ type }} {{ item.comment }} from {{ item.from }} ip on {{ item.proto }}"`